src/tests/ftest/util/daos_utils_base.py

```diff
@@ -268,8 +268,7 @@ def __init__(self):
                 #   --acl-file=PATH
                 #           input file containing ACL
                 self.acl_file = FormattedParameter("--acl-file={}", None)
-                #     -l, --label=<container label>
-                self.label = FormattedParameter("--label={}")
+                self.label = FormattedParameter("{}", None)
```

Comment:
This technically works, but to be safe:
```suggestion
                self.label = PositionalParameter(3)
```
"3" because it's after "2" on line 215 above. (the test code usage of `cont` (line 215) and `label` (line 271) should be consolidated, but that's outside the scope of your PR)

Reply:
Ah, right. I didn't think that my approach was 100% correct but I didn't see `PositionalParameter`. I'll fix that. Will hold off a bit to see if there are any other comments first though.

Reply:
Positional parameters are usually placed first, then other "--" parameters, so it's better to move this line to the top of the method to be consistent with the command and other methods.